_config.yml

```diff
@@ -8,7 +8,7 @@ logo: '/public/logo.png'
 highlighter: rouge
 
 # Site configurations
-gems:
+plugins:
```

Comment:
Not sure if this will work with Github pages version of Jekyll but we can give it a try

Reply:
The term "gems" is now deprecated. GitHub Action when building Jekyll is the one actually warned me about it.
> Deprecation: The 'gems' configuration option has been renamed to 'plugins'. Please update your config file accordingly.